content/200-concepts/300-more/050-environment-variables/100-managing-env-files-and-setting-variables.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: 'Managing .env files and setting variables'
+metaTitle: 'Managing .env files and setting variables'
+metaDescription: 'Learn how to manage .env files and set environment variables'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Although [Prisma creates an `.env` file for you upon installation](./#how-does-prisma-use-environment-variables), there can be times when you want to:
```

Comment:
Where does this link to?

Reply:
See https://github.com/prisma/docs/pull/1860#discussion_r663699265